fw-child/template/post-meta.php

```diff
@@ -1,57 +1,59 @@
-<div class="news-meta-item mb-3 mb-lg-4">
-	<h6 class="mb-3 all-caps text-blue-100"><?php _e ( 'Date', 'cdc' ); ?></h6>
-	
-	<?php 
-		
-		the_time ( __( 'F j, Y', 'cdc' ) );
-	
+<div class="news-meta-block p-3 border border-gray-600">
+	<div class="news-meta-item mb-3 mb-lg-4">
+		<span class="mb-1 all-caps text-blue-100 d-block"><?php _e ( 'Date', 'cdc' ); ?></span>
+
+		<?php
+
+			the_time ( __( 'F j, Y', 'cdc' ) );
+
+		?>
+	</div>
+
+	<div class="news-meta-item mb-3 mb-lg-4">
+		<span class="mb-1 all-caps text-blue-100 d-block"><?php _e ( 'Author', 'cdc' ); ?></span>
+
+		<?php
+
+			the_field ( 'post_author' );
+
+		?>
+	</div>
+
+	<?php
+
+		$post_tags = get_the_terms ( get_the_ID(), 'post_tag' );
+
+		if ( !empty ( $post_tags ) ) {
+
 	?>
-</div>
 
-<div class="news-meta-item mb-3 mb-lg-4">
-	<h6 class="mb-3 all-caps text-blue-100"><?php _e ( 'Author', 'cdc' ); ?></h6>
-	
+	<div class="news-meta-item mb-3 mb-lg-4">
+		<span class="mb-1 all-caps text-blue-100 d-block"><?php _e ( 'Topics', 'cdc' ); ?></span>
+
+		<p><?php
```

Comment:
The hover effect of those "Topics" links is not the same as the hover effect on the social links. It should be uniformized. Note: it can be done simply by changing this `<p>` with a `<div>` (just validate that the margins are still OK after change). 

(I know the hover effect is probably too subtle, but it can be made into its own task later on, since the same effect is applied in various parts of the site)

```suggestion
		<div><?php
```

Reply:
@xfrenette-crim this has been fixed, there is no need for `<div>` as well.